app/controllers/admin/users_controller.rb

```diff
@@ -387,10 +391,5 @@ def set_current_tab(current_tab = "overview")
                        "overview"
                      end
     end
-
-    def set_banishable_user
```

Comment:
Moved this to a helper instead so that we can reuse this between the index and show page. It also seems like a view layer concern hence a helper made sense.